src/main/java/bio/terra/service/dataset/flight/ingest/IngestInsertIntoDatasetTableStep.java

```diff
@@ -49,6 +49,12 @@ public StepResult doStep(FlightContext context) throws InterruptedException {
       BulkLoadArrayResultModel fileLoadResults =
           workingMap.get(IngestMapKeys.BULK_LOAD_RESULT, BulkLoadArrayResultModel.class);
       ingestResponse.loadResult(fileLoadResults);
+      Long failedRowCount = 0L;
```

Comment:
If a value can't be `null` I think an unboxed type is better because it more closely represents what the code's intent is. Changing it also lets you remove the `L`.
```suggestion
      long failedRowCount = 0;
```